ECOSYSTEM.md

```diff
@@ -14,12 +14,19 @@ Partners are companies that are significantly invested in React Native and demon
 
 Partners think of React Native as a product; they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications.
 
-Baseline responsibilities of partners include:
-* Attending monthly meeting
-* Contributing to the release process. Examples include being a [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser), testing new releases, technical support for release issues.
-* Engage on the core contributor Discord
+### Application process
+To become a React Native partner, an existing partner needs to refer and champion your application. Partners will undergo a 3 month incubating period, after which Partners will vote to convert to full membership.
 
-Our current partners and other areas of ownership:
+Partnership is not a status symbol, it is a commitment to invest significant resources into improving React Native.
+
+Maintaining partner status requires consistently meeting the baseline responsibilities, including:
+* Attending monthly meeting.
+* Contributing to the release process.
+* Engaging in the core contributor Discord.
```

Comment:
```suggestion
Maintaining partner status requires consistently meeting the baseline responsibilities, including:
* Attending monthly meeting.
* Contributing to the release process.
* Engaging in the core contributor Discord.
* Meaningfully contribute to the ecosystem and the broader community
```

since most existing partners do very important things (ex IR and Callstack and Expo+SWM doing the confs) for the broader community it might be worth to mention? Like more partners do support the broader community than the ones contributing to the release process, so might be worth to have some sort of mention there for perspective partners to consider themselves "reaching the bar" for applying

Reply:
I agree with @kelset. Maybe not all partners should contribute to the release process but have meaningful contributions to the ecosystem. I think a good idea is to add examples of meaningful contributions (doesn't need to be concrete) as the definition is very vague. We mentioned in the meeting that it could be contributions to core and popular libraries but also conferences and podcasts.

Reply:
Great points. I didn't change any of these here though, just changed the formatting, so let's followup to up if we want to change the baseline responsibilities.